pmm-app/src/pmm-qan/panel/provider/provider.tools.ts

```diff
@@ -15,67 +14,68 @@ const setFilters = (query) => FILTERS_NAMES.reduce((acc, filterName) => {
   return acc;
 }, {});
 
+interface GrafanaVariables {
```

Comment:
Saw these changes were introduced on the other PR as well, I think it's better to merge the other first and rebase on top of PMM-2.0 so no need to introduce these changes here as well

Reply:
💯 This way there will be less to review.

Reply:
Agree, but we need to wait for grafana7 and merge it only then